docs/sources/enterprise/query-caching.md

```diff
@@ -21,20 +21,27 @@ The following cache backends are available: in-memory, Redis, and Memcached.
 - Reduced API costs.
 - Reduced likelihood that APIs will rate-limit or throttle requests.
 
-## Enable query caching
+## Enable and configure query caching
 
-> **Note:** The query caching feature is enabled by default. To start caching data source queries, visit the data source's configuration page.
+By default, data source queries are not cached. To enable query caching for a single data source:
+1. Visit the data source settings page.
```

Comment:
Rewrite these top three steps to match the comments in Disable query caching.

Reply:
1. On the side menu, click...
2. In the data source list, click the data source that you want to turn off caching for.
3. In the Cache tab, click Disable.